external-crates/move/crates/move-binary-format/src/compatibility.rs

```diff
@@ -4,14 +4,17 @@
 
 use std::collections::BTreeSet;
 
+use crate::normalized::{Enum, Function, Struct};
 use crate::{
```

Comment:
Uber nit, and I am guilty of this too (thanks to Rust analyzer): but might be nice to merge the different `use` statements here.